client/src/components/Login.vue

```diff
@@ -1,34 +1,59 @@
 <template>
-  <section>
-    <v-dialog v-model="dialog" :fullscreen="mobile" max-width="500px">
-      <template v-slot:activator="{ props }">
-        <v-btn v-bind="props">Login</v-btn>
-      </template>
-      <v-card>
-        <form @submit.prevent="submitForm">
-          <v-card-title>
-            <span class="text-h5">Login</span>
-          </v-card-title>
-          <v-card-text>
+  <v-button>
+  <v-model fullscreen :scrim="false">
+  <v-col>
```

Comment:
im not sure what the vuetify v-model element is. v-dialog is the correct element for modals.